blog/en-us/DS_run_in_windows.md

```diff
@@ -0,0 +1,102 @@
+# Set-up DolphinScheduler Development Environment and Run Source Code on Windows OS
+
+1. ## Download Source Code
+
+   Official Website：https://dolphinscheduler.apache.org/en-us/index.html
+
+   GitHub Repository：https://github.com/apache/dolphinscheduler.git
+
+   Here we use 1.3.8-release tag.
+
+2. ## Install Zookeeper on Windows
+
+   1. Download zookeeper https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3-bin.tar.gz
+
+   2. Unzip 'apache-zookeeper-3.6.3-bin.tar.gz'
+
+   3. Create 'data' and 'log' folder under zookeeper directory
+
+   4. Copy the 'zoo_sample.cfg' file in the 'conf' directory, rename it to 'zoo.cfg' and modify the data and log directory settings:
+
+      ```shell
```

Comment:
It's better to use `~~~` here, consistent with following

Reply:
It’s not `shell` syntax, should be `properties`

Reply:
done